src/main/java/iudx/resource/server/apiserver/management/ManagementApiImpl.java

```diff
@@ -448,19 +498,21 @@ public Future<JsonObject> publishDataFromAdapter(JsonObject json, DataBrokerServ
   }
 
   @Override
-  public Future<JsonObject> publishAllAdapterForUser(JsonObject request, DataBrokerService databroker) {
+  public Future<JsonObject> publishAllAdapterForUser(JsonObject request, PostgresService postgresService) {
```

Comment:
The method name `getAllAdapterDetailsForUser` makes more sense than `publishAllAdapterForUser`. Change method name to `getAllAdapterDetailsForUser`